src/settings/index.ts

```diff
@@ -614,6 +616,14 @@ export const ALLOWED_EMPTY_TABLES_NUMBER = 4;
 export const DEFAULT_JAZZBAR_TABLES_NUMBER = 12;
 export const DEFAULT_CONVERSATION_SPACE_TABLES_NUMBER = 10;
 
+export const JAZZBAR_TABLES: Table[] = generateTables({
+  num: DEFAULT_JAZZBAR_TABLES_NUMBER,
+});
+
+export const CONVERSATION_TABLES: Table[] = generateTables({
+  num: DEFAULT_CONVERSATION_SPACE_TABLES_NUMBER,
+});
+
```

Comment:
```suggestion

// These are really supposed to be constants and to avoid possible mutable shared state in the code elsewhere
Object.freeze(JAZZBAR_TABLES);
Object.freeze(CONVERSATION_TABLES);

```

Reply:
done 2ba4481